### PR TITLE
my first card

### PR DESCRIPTION
**Pull request by kalabYibeltal**

### CODE REVIEW

File: starter-project-web-react/pages/blogs.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+              },      
+              {
+                "title": "good man!",
+                "body" : "Lorem ipsum dolor sit amet consectetur adipisicing elit. Assumenda a rerum quibusdam eligendi doloribus esse aspernatur sit dolorum? Id perferendis excepturi officia quia. Beatae porro natus aliquam deleniti perferendis illum!",
+                "author": "yoshi",
+                "id": 2
+              }, 
+        ]
+    return (  
+        <div className="blog-list">
```
Comment (YiseBoge) [inline starter-project-web-react/pages/blogs.tsx:37]: Is this using material UI or just plain CSS?

**Review by YiseBoge — CHANGES_REQUESTED** [review]
Looks good overall, but I would say the commit message (hence the name of the PR) is very good
You can rename it to make it an imperative sentence like "create a blog list page" or sth, and also include the scope of the commit (which is sth like [web]) at the beginning of the commit message because the commit is viewed be all members in the repo (refer to the instructions in the root folder readme.md)
I would also recommend you rename the branch in the form or [your-name].[feature]

**Comment by YiseBoge** [comment]
One more comment...
It is better to add page in a "blog" folder and make it the index.tsx (to be routed as /blog) file in there because there might be other pages under there (like "/blog/id", "/blog/new" or sth like that)

File: starter-project-web-react/pages/blogs.tsx (hunk 1 above)
Comment (kalabYibeltal) [reply starter-project-web-react/pages/blogs.tsx:37]: plain css for now, but i am going to look into that.

**Pull request merged**